### ISSUE TITLE
Create CiviCRM Class

### ISSUE THREAD

**Issue by elyse-weiss** [move-coop/parsons#70]
Some helpful existing documentation: https://github.com/4ndygu/civicrm_osdi

**Comment by joemcl** [comment]
hi and thanks @felipealbertao , will check out the snippet. 

Also noting here that CiviCRM has an API, that concern was raised in a meeting just now...

**Comment by felipealbertao** [comment]
I've been exploring CiviCRM's API, and I'm impressed. It's very well thought out and easy to use.

If we can make CiviCRM more integrated with other tools like Spoke, CiviCRM has a huge potential to serve as an alternative to expensive, closed-source and anti-democratic election solutions like PDI and VAN

**Comment by joemcl** [comment]
There is a newish integration with Spoke, I'll post the link here. It doesn't use a Parsons connector.

**Comment by johntwyman** [comment]
Yep, we have a working CiviCRM-Spoke integration up and running. We're making extensive use of it at the moment.

The Pull Request needs another update and a final bit of polish. But the code is ready to use if people wish.

Grab the `contact-loaders/civicrm` directory and the relevant `action-handlers` from [here](https://github.com/australiangreens/Spoke/tree/main/src/extensions) if you're interested in giving it a spin.

**Comment by joemcl** [comment]
Hi @johntwyman @felipealbertao might y'all be available today at 6:00 p.m. US Eastern Time/ 3:00 p.m. US Pacific Time/ 8:00 a.m. Sydney time?

**Comment by felipealbertao** [comment]
I'm available at 3p PST

**Comment by joemcl** [comment]
Great @felipealbertao , are you on the progcode Slack yet?

**Comment by joemcl** [comment]
If not, please sign up here - https://progco.de/join

**Comment by felipealbertao** [comment]
Where are we meeting? I have only until 3:30p PST

**Comment by joemcl** [comment]
@felipealbertao I'm sorry, let's catch up on the other chat thread
